<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPedidosCliente</class>
 <widget class="QDialog" name="TelaPedidosCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Caviar Dreams</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabProd">
    <attribute name="title">
     <string>Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="twCompraCliente">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>751</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAdiciona">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,60);
	color:white;
	border-style: solid;
	border-width:3px;
	border-color: white;
	border-radius: 6px;

}
QPushButton::hover{
	background-color:rgba(255,255,255,60);
	color:black;
	border-style: solid;
	border-width:4px;
	border-color: black;
	border-radius: 6px;

}</string>
     </property>
     <property name="text">
      <string>Adicionar Produto a Compras</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonListar">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,60);
	color:white;
	border-style: solid;
	border-width:3px;
	border-color: white;
	border-radius: 6px;

}
QPushButton::hover{
	background-color:rgba(255,255,255,60);
	color:black;
	border-style: solid;
	border-width:4px;
	border-color: black;
	border-radius: 6px;

}</string>
     </property>
     <property name="text">
      <string>Listar todos os produtos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPesquisar">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,60);
	color:white;
	border-style: solid;
	border-width:3px;
	border-color: white;
	border-radius: 6px;

}
QPushButton::hover{
	background-color:rgba(255,255,255,60);
	color:black;
	border-style: solid;
	border-width:4px;
	border-color: black;
	border-radius: 6px;

}</string>
     </property>
     <property name="text">
      <string>Pesquisar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pesquisar Produto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPesquisar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Digite o nome do produto</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnQuantidade">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="usefulResources.qrc">:/src/bgAfter/a_Gerencia 1.png</pixmap>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>twCompraCliente</zorder>
    <zorder>buttonAdiciona</zorder>
    <zorder>buttonListar</zorder>
    <zorder>buttonPesquisar</zorder>
    <zorder>label</zorder>
    <zorder>txtPesquisar</zorder>
    <zorder>spnQuantidade</zorder>
   </widget>
   <widget class="QWidget" name="tabComp">
    <attribute name="title">
     <string>Compras</string>
    </attribute>
    <widget class="QTableWidget" name="twPedidosCliente">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConfirma">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,60);
	color:white;
	border-style: solid;
	border-width:3px;
	border-color: white;
	border-radius: 6px;

}
QPushButton::hover{
	background-color:rgba(255,255,255,60);
	color:black;
	border-style: solid;
	border-width:4px;
	border-color: black;
	border-radius: 6px;

}</string>
     </property>
     <property name="text">
      <string>Confirmar pedido de compras</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,60);
	color:white;
	border-style: solid;
	border-width:3px;
	border-color: white;
	border-radius: 6px;

}
QPushButton::hover{
	background-color:rgba(255,255,255,60);
	color:black;
	border-style: solid;
	border-width:4px;
	border-color: black;
	border-radius: 6px;

}</string>
     </property>
     <property name="text">
      <string>Excluir pedido de minhas compras</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>781</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="usefulResources.qrc">:/src/bgAfter/a_Pedidos.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>twPedidosCliente</zorder>
    <zorder>buttonConfirma</zorder>
    <zorder>pushButton</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="usefulResources.qrc">:/src/bgAfter/a_Gerencia 1.png</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources>
  <include location="usefulResources.qrc"/>
 </resources>
 <connections/>
</ui>
